### DISCUSSION TITLE
FP: format of node text shortcut

### DISCUSSION THREAD

**Discussion by Illegit** [freeplane/freeplane#1802] [Help] [answered]
Hi I ask how is possible to format the node text (e.g. dates), in case of dates in some cases the command format( some_date,'yyyy-MM-dd') is not working
![image](https://github.com/freeplane/freeplane/assets/62674157/0399dd66-1448-48ac-8a3d-818aa1507efb)
in the image the dates are created with the previous command but when the date has been generated by a formula with the previous command the result is a date with full time even, but I like to place there just the normal date. I was looking for a FP command showing this format but it is only in the panel --->> here my final question: how to activate it with a shortcut?

**Comment by Illegit** [comment]
How to transform this command in a script? 
![image](https://github.com/freeplane/freeplane/assets/62674157/998adb63-9755-4e7a-abe6-8fd6af5f4a88)

**Comment by euu2021** [comment, marked as answer]
Maybe, an idea is to create a conditional style that applies this kind of Format when the node content matches a certain filter criteria.
